### ISSUE TITLE
Accessibility Issue with Side bars - via Keith

### ISSUE THREAD

**Issue by sydbeth** [ATLAS-Illinois/Illinois-Framework-Theme#1020]
**Describe the bug**
Complementary regions (aside with a label or role=complementary) must be top-level regions (e.g. they cannot be inside any other landmark region).

**To Reproduce**
1. use a region scanner like bookmarklets to see complementary region inside of main region.

**Solutions**

Make the aside tag a div tag and remove role = complementary

OR

Move the aside to a top-level item outside of the main tag
